src/static/libs/js/mediaPlayer.js

```diff
@@ -1,299 +1,30 @@
-/*eslint-disable*/
-'use strict';
+import { calcDuration } from './utility.js';
+import { convertSecToHHMMSS } from './utility.js';
```

Comment:
A lot of these can be reduced to

`import { calcDuration, convertSecToHHMMSS, isonAnonLoggedInUser } from './utility.js' `